### PR TITLE
Add URLs for recipients

### PR DESCRIPTION
**Pull request by qubist**

### CODE REVIEW

**Comment by qubist** [comment]
will resolve #55

**Comment by lbraun** [comment]
Good start Will. 

Tips: you will want to create a "migration" to add this field to the database. Read about migrations here:
https://edgeguides.rubyonrails.org/active_record_migrations.html

You will also want to add the url field to the form, make sure it is one of the permitted attributes in the controller, and potentially add it to the index too. Let me know if you have any questions!

**Comment by qubist** [comment]
@lbraun Thanks for the link! I've made the changes and they pass my test from earlier. Would you have a look?

One ongoing issue is that on the Edit Recipient page the "URL" label is displayed weirdly: 
Screen Shot 2020-08-11 at 7 30 28 PM

I'd like it instead to be "URL". I think this has to do with humanize. Fixing this may be another issue though, as it also might overlap with the fix for issue #2 changing the text for credit_card to "Card"

**Comment by lbraun** [comment]
Yeah we should use `human_attribute_name` instead of `humanize`:
https://guides.rubyonrails.org/i18n.html#translations-for-active-record-models

Translations are stored in https://github.com/lbraun/thanksgiving/tree/main/config/locales

That's kind of a weird structure I started for them there, we could switch to having just one file for all english translations. But translations is definitely the way to handle this.

File: db/migrate/20200811231207_add_url_to_recipients.rb [hunk 1]
```diff
@@ -0,0 +1,5 @@
+class AddUrlToRecipients < ActiveRecord::Migration[5.2]
+  def change
+    add_column :recipients, :url, :string
```
Comment (lbraun) [inline db/migrate/20200811231207_add_url_to_recipients.rb:3]: Perfect! 💯

**Comment by qubist** [comment]
Ready to be merged!

**Pull request merged**